packages/react/src/SubNav/SubNav.tsx

```diff
@@ -267,26 +469,108 @@ const SubNavLink = forwardRef<HTMLAnchorElement | HTMLDivElement, SubNavLinkProp
     <li>
       <a
         href={href}
```

Comment:
I'm not sure what the ideal behaviour is here, but at the moment activating an anchor nav link scrolls the page but doesn't move focus. This mean that a screen reader user, or a keyboard user, can activate a link however when they try to navigate to the next element they will still be inside this list of anchor links, which is counter-intuitive.

I'd imagine that we'd also want to move focus to the element that the anchor nav is linking to, wdyt?

Reply:
Thanks 👍, I've removed the `scrollTo` and reverted to native browser behaviour to resolve this. Now works the same as AnchorNav, which should be correct.